datatree/tests/test_formatting.py

```diff
@@ -108,13 +108,13 @@ def test_diff_node_data(self):
         Data in nodes at position '/a' do not match:
 
         Data variables only on the left object:
-            v        int64 1
+            v        int64 8B 1
 
         Data in nodes at position '/a/b' do not match:
 
         Differing data variables:
-        L   w        int64 5
-        R   w        int64 6"""
+        L   w        int64 8B 5
+        R   w        int64 8B 6"""
```

Comment:
This difference is due to a newer version of xarray - [v2024.2.0](https://docs.xarray.dev/en/stable/whats-new.html#v2024-02-0-feb-19-2024). Technically I guess we should pin to that version to match this change?